tests/test_study.py

```diff
@@ -437,3 +438,77 @@ def f(trial):
             assert df.params.y[i] == 2.5
             assert df.user_attrs.train_loss[i] == 3
             assert ('system_attrs', 'fail_reason') in df.columns
+
+
+@pytest.mark.parametrize('storage_mode', STORAGE_MODES)
+def test_create_study(storage_mode):
+    # type: (str) -> None
+
+    with StorageSupplier(storage_mode) as storage:
+        # Test creating a new study.
+        study = optuna.create_study(storage=storage, exist_ok=False)
+
+        if not isinstance(study.storage, optuna.storages.InMemoryStorage):
+            # Test exist_ok=True with existing study.
+            optuna.create_study(study_name=study.study_name, storage=storage, exist_ok=True)
+
+            # Test exist_ok=False with existing study.
+            with pytest.raises(optuna.structs.DuplicatedStudyError):
+                optuna.create_study(study_name=study.study_name, storage=storage, exist_ok=False)
+
+
+def test_create_study_parallel():
```

Comment:
Does this unit test assume simultaneous creation of study? If so, CPython's GIL will prevent the concurrent execution of threads.

Reply:
> Does this unit test assume simultaneous creation of study? 

Yes.
I didn't know about GIL...
I will fix this test case (or maybe I will remove this).

Reply:
I removed this test because the current test is meaningless and testing parallel execution feels to be too costly.